packages/app/background-jobs-common/src/background-job-processor/monitoring/BackgroundJobProcessorMonitor.ts

```diff
@@ -89,19 +90,33 @@ export class BackgroundJobProcessorMonitor<
   public jobStart(job: JobType, requestContext: RequestContext): void {
     const transactionName = `bg_job:${this.ownerName}:${this.queueId}`
     this.transactionObservabilityManager.start(transactionName, resolveJobId(job))
-    requestContext.logger.info({ origin: this.processorName }, `Started job ${job.name}`)
+    requestContext.logger.info(this.buildLogParams(job), `Started job ${job.name}`)
+  }
+
+  public jobTryError(job: JobType, error: unknown, requestContext: RequestContext): void {
+    requestContext.logger.error(this.buildLogParams(job, error), `${job.name} try failed`)
   }
 
   public jobEnd(job: JobType, requestContext: RequestContext): void {
     requestContext.logger.info(
       {
-        origin: this.processorName,
+        ...this.buildLogParams(job),
         isSuccess: job.progress === 100,
       },
       `Finished job ${job.name}`,
     )
     this.transactionObservabilityManager.stop(resolveJobId(job))
   }
+
+  private buildLogParams(job: JobType, error?: unknown) {
+    return {
+      origin: this.processorName,
```

Comment:
wonder if more of these fields should be moved to the childLogger building step

Reply:
Not really, origin is where is the log generated, if the request context is passed to a service/use case, origin should not be there from my point of view, but please feel free to disagree

Reply:
I don't mean the requestContext that we pass any deeper, just the logger that we instantiate for the job. I mean this part:

```ts
    const requestContext: RequestContext = {
      reqId: jobId,
      logger: new BackgroundJobProcessorLogger(
        this.logger.child({
          // add more stuff here, it won't propagate anywhere else
          'x-request-id': job.data.metadata.correlationId,
          jobId,
          jobName: job.name,
        }),
        job,
      ),
    }
```

Reply:
Yeah, but we should always use `requestContext.logger`, right?
If, from the `processor.process` method, we call to a use-case or service where we need to log something we should be passing `requestContext` and in that case origin is not correct

Reply:
ah, fair point.
but isn't that already a problem?

Reply:
for other properties, it is ...  well I guess it's fine to put the job id or job name to see which was the "starter" of the process but we are using `origin`  to put the class from where the log is written